src/output.rs

```diff
@@ -83,31 +83,39 @@ impl OutputType {
             let replace_arguments_to_less = pager.source == PagerSource::EnvVarPager;
 
             if args.is_empty() || replace_arguments_to_less {
-                p.arg("--RAW-CONTROL-CHARS");
```

Comment:
If busybox less understands `-R` but not `--RAW-CONTROL-CHARS`, it would IMO make sense to change

```
p.arg("--RAW-CONTROL-CHARS");
```

into 

```
p.arg("-R"); // Short version of --RAW-CONTROL-CHARS to maximize compatibility
```

Same for `--quit-if-one-screen` and `-F`.

Does busybox have a short variant of `--chop-long-lines`? If so that would be great, because then we could avoid moving this block of code down, and instead reuse the logic for both buxybox less and less proper. The old code is like bomb waiting to explode with regressions, so it would be great if we could try to stick to "obviously safe" changes, with a minimal "structural" diff.

Reply:
> Does busybox have a short variant of --chop-long-lines?
Yes, and I just checked that it is the same short option as regular less. So that's three options that could be passed the same way to both. Definitely good points.